com.unity.render-pipelines.high-definition/Runtime/Material/Material.hlsl

```diff
@@ -76,23 +80,29 @@ float4 EvaluateAtmosphericScattering(PositionInputs posInput, float3 V, float4 i
     float3 volColor, volOpacity;
     EvaluateAtmosphericScattering(posInput, V, volColor, volOpacity); // Premultiplied alpha
 
-    #if defined(_BLENDMODE_ALPHA)
+    if (_BlendMode == BLENDMODE_ALPHA)
```

Comment:
we shoud simplify those branch

result.rgb = result.rgb * (1.0 - volOpacity) + (_BlendMode != BLENDMODE_ADDITIVE) ? volColor * result.a : 0.0;

Reply:
I did think about it, but did not to mantain the lengthy comments there and the actual cost is fairly small. 

I guess I can reword the comments, we'd lose a bit of clarity, but should still be ok.

Reply:
you can just comment all the current code and put an optimized version at the end, it is fined :)

Reply:
To be clear though, both version compile to exactly the same amount of ops/cycles as it recognizes the two branches are the same. 

The scalar branch and the ternary operator both translate to a v_cmp_eq_f32 anyway.